Apps/WebClient/src/ClientApp/src/components/navmenu/sidebar.vue

```diff
@@ -171,32 +175,46 @@ export default class SidebarComponent extends Vue {
         this.eventBus.$emit(EventMessageName.TimelineCreateNote);
     }
 
-    private dismissTutorial() {
-        this.logger.debug("Dismissing tutorial...");
-        if (
-            this.user.preferences.tutorialPopover != undefined &&
-            this.user.preferences.tutorialPopover.hdId != undefined
-        ) {
-            this.user.preferences.tutorialPopover.value = "false";
+    private dismissTutorial(
+        preference: string,
+        userPreference: UserPreference
```

Comment:
i don't think you need to pass both values, this.user.preferences[preference] should give you the UserPreference

Reply:
aha nice suggestion!

Reply:
code updated & tested